napari/layers/image/_slice.py

```diff
@@ -108,11 +112,18 @@ def make_empty(
             If None, a new request id will be returned, which guarantees that
             the empty slice never appears as loaded. (Used for layer
             initialisation before attempting data loading.)
+        dtype : np.dtype
+            The dtype of the empty image slice.
         """
         shape = (1,) * slice_input.ndisplay
+        if dtype is None:
+            dtype = np.uint8
```

Comment:
Is there a compelling reason to allow None? Maybe set the default dtype to np.uint8?

Reply:
mypy

Reply:
yeah but mypy wants it because that's how we're using it, right? We can fix it all the way across?

Reply:
should be done now